README.md

```diff
@@ -43,16 +42,27 @@ test('Add new item', async t => {
 #### Obtaining component's props, computed and state
 
 In additional to [testcafe snapshot properties](http://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/dom-node-state.html), you can obtain `state`, `computed` or `props` of a Vue component. You can use them in an assertion directly thus simplifying assertion logic.
```

Comment:
I think it's better to change `testcafe snapshot properties` to `DOM Node State` properties since it was changed in the testcafe documentation